aea/helpers/search/models.py

```diff
@@ -108,6 +110,34 @@ def __iter__(self):
         """Create an iterator."""
         return iter(self.values)
 
+    @classmethod
+    def encode(cls, performative, description_from_message: "Description"):
+        """
+        Encode an instance of this class into the protocol buffer object.
+
+        The content in the 'performative' argument must be matched with the message content in the 'description_from_message' argument.
+
+        :param performative: the performative protocol buffer object containing a content whose type is this class.
+        :param description_from_message: the message content to be encoded in the protocol buffer object.
+        :return: the 'performative' protocol buffer object encoded with the message content in the 'description_from_message' argument.
+        """
+        description_from_message_bytes = pickle.dumps(description_from_message)  # nosec
```

Comment:
Imperfect, but a temporary working solution. Eventually, we make these serializable with protobuf and remove the dependency on performative.